src/components/Avatar/AvatarGroup.stories.tsx

```diff
@@ -202,9 +202,10 @@ export const Basic = Basic_Story.bind({});
 
 export const Basic_Spaced = Basic_Story.bind({});
 
+export const Basic_Max_Props_Exceed_Children = Basic_Story.bind({});
+
 const List_Story: ComponentStory<typeof AvatarGroup> = (args) => (
```

Comment:
nit: Update `maxProps` `tooltipProps.content` to something that matches the UI, like `User 6, User 7` or simply 'This is a tooltip.`.

Reply:
doing.

Reply:
done.